src/user-icon/user-icon.scss

```diff
@@ -4,57 +4,47 @@
 @import '../style/index';
 @import '../text/mixins';
 
-@mixin iui-user-icons($size: $iui-icons-xl) {
+@mixin iui-user-icon {
```

Comment:
So this mixin doesn't take size anymore. Were we using the size parameter anywhere else or just in the size classes?

Reply:
I'm not aware of it being used anywhere else.